app/controllers/users_controller.rb

```diff
@@ -250,7 +250,7 @@ def validate_update(params)
     if params[:username].nil? || params[:username].length < 1
       errors << "Last name field is required"
     end
-    if ((!params[:orcidId].match(/^\d{4}+(-\d{4})+$/)) || (params[:orcidId].length != 19)) && !(params[:orcidId].nil? || params[:orcidId].length < 1)
+    if params[:orcidId].present? && ((!params[:orcidId].match(/^\d{4}-\d{4}-\d{4}-\d{4}$/)) || (params[:orcidId].length != 19))
```

Comment:
you changed the REGEX, why ?

Reply:
because a valid ORCID ID consists of 16 digits and 3 dashes

Reply:
@Bilelkihal please review this.